checker/jdk/signedness/src/java/lang/Integer.java

```diff
@@ -1274,7 +1274,7 @@ public static int compareUnsigned(@Unsigned int x, @Unsigned int y) {
      *         conversion
      * @since 1.8
      */
-    public static @Unsigned long toUnsignedLong(@UnknownSignedness int x) {
+    public static @SignedPositive long toUnsignedLong(int x) {
```

Comment:
@mernst Why we have annotated this as @SignedPositive?

Reply:
Because it is always a positive value.  More specifically, it is a long value in the range 0..2^32-1.
When you ask a question, please give context.  If you believe that the `@SignedPositive` annotation is wrong, then say that *explicitly* so that we can have a conversation, rather than making me guess about why you are asking the question.  Please clarify.

Reply:
Because it was converting the integer to an unsigned as the method is `toUnsignedLong` thats why I thought that it should be annotated as `@Unsigned`. But now I got your point.

Reply:
@mernst Due to this change `UtilsJava8.java` test was failing. I have updated the test case and the jdkHash.
Now you can continue with your review.